cmd/format-xl.go

```diff
@@ -753,16 +754,43 @@ func fixFormatXLV3(storageDisks []StorageAPI, endpoints Endpoints, formats []*fo
 func initFormatXL(ctx context.Context, storageDisks []StorageAPI, setCount, drivesPerSet int, deploymentID string) (*formatXLV3, error) {
 	format := newFormatXLV3(setCount, drivesPerSet)
 	formats := make([]*formatXLV3, len(storageDisks))
-
+	logger.Info("Formatting Zone...")
 	for i := 0; i < setCount; i++ {
+		logger.Info(" * Set %v:", i+1)
+		hostCount := make(map[string]int, drivesPerSet)
 		for j := 0; j < drivesPerSet; j++ {
+			disk := storageDisks[i*drivesPerSet+j]
 			newFormat := format.Clone()
 			newFormat.XL.This = format.XL.Sets[i][j]
 			if deploymentID != "" {
 				newFormat.ID = deploymentID
 			}
+			logger.Info("   - Drive: %s", disk.String())
```

Comment:
Above zone format logging is fine but logging Drives will be too much on our console, we are deploying like 2000 drives in single setup it will  look baffling. Perhaps can say N drives on this set formatted. Only if we need to log the offending node we can log all the drives and host information.